README.md

```diff
@@ -52,7 +52,7 @@ php app/console  doctrine:migrations:migrate
 in the vagrant machine (in directory `/vagrant` ) run: (in case you want to use the grant type `password`)
 
 ```
-php app/console acme:oauth-server:client:create --grant-type="password" --grant-type="refresh_token" --grant-type="token"
+php app/console acme:oauth-server:client:create --grant-type="password" --grant-type="refresh_token" --grant-type="token" --client-type="backend"
```

Comment:
precise the --client-type is optional , and add that client-type "backend" is a value that will be used for permission to some of the api of this oauth2